csharp/src/Apache.Arrow.Adbc/Apache.Arrow.Adbc.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Version>0.1.0</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="Extensions\MatchCollectionExtensions.netstandard.cs" />

```

Comment:
Does this need a `Condition=TFM == netstandard` on it?

Reply:
no. this was removed.